tests/test_amqp.py

```diff
@@ -1597,7 +1601,7 @@ async def test_connection_close(
         try:
             with pytest.raises(aio_pika.exceptions.ChannelPreconditionFailed):
                 msg = Message(bytes(shortuuid.uuid(), "utf-8"))
-                msg.delivery_mode = 8
+                msg.delivery_mode = 8       # type: ignore
```

Comment:
what's happened here? why mypy is struggling?

Reply:
Because it's IntEnum